src/test/java/com/google/gcp_variant_transforms/task/VcfToBqTaskTest.java

```diff
@@ -0,0 +1,112 @@
+// Copyright 2020 Google LLC
+
+package com.google.gcp_variant_transforms.task;
+
+import static com.google.common.truth.Truth.assertThat;
+import static org.mockito.Mockito.doNothing;
+import static org.mockito.Mockito.doReturn;
+import static org.mockito.Mockito.spy;
+import static org.mockito.Mockito.verify;
+
+import com.google.common.collect.ImmutableList;
+import com.google.gcp_variant_transforms.beam.PipelineRunner;
+import com.google.gcp_variant_transforms.library.HeaderReader;
+import com.google.gcp_variant_transforms.options.VcfToBqContext;
+import com.google.gcp_variant_transforms.options.VcfToBqOptions;
+import com.google.inject.Injector;
+import java.io.IOException;
+import org.apache.beam.sdk.options.PipelineOptions;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Captor;
+import org.mockito.MockitoAnnotations;
+import org.mockito.Mock;
+
+/**
+ * Units tests for VcfToBqTask.java
+ */
+ public class VcfToBqTaskTest {
+  Injector spyInjector;
+  VcfToBqTask vcfToBqTask;
+  VcfToBqTask spyVcfToBqTask;
+
+  @Mock
```

Comment:
I think we can inline, just to save space stylistically:

@Mock PipelineRunner mockedPipelineRunner.

Here and below.